src/v/cluster/partition_balancer_backend.cc

```diff
@@ -38,6 +38,7 @@ partition_balancer_backend::partition_balancer_backend(
   ss::sharded<health_monitor_frontend>& health_monitor,
   ss::sharded<partition_allocator>& partition_allocator,
   ss::sharded<topics_frontend>& topics_frontend,
+  ss::sharded<members_table>& members_table,
```

Comment:
we can just use `state.members()` everywhere in the balancer, no need to pass another reference separately.